src/wallet/wallet.ts

```diff
@@ -322,8 +322,8 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     let privKeyAccountPath: bitcore.HDPrivateKey;
     let authDerivedPrivKey: bitcore.HDPrivateKey;
 
-    const now = new Date();
-    const timestampNow = Math.floor(now.getTime() / 1000); // in seconds
+    const now = Date.now();
+    const timestampNow = Math.floor(now / 1000); // in seconds
```

Comment:
Was this changed so we could mock `Date.now()` on the tests?

Reply:
Exactly